docs/utilities/tags.md

```diff
@@ -9,13 +9,24 @@ Tags are declared in your mod's [datapack][datapack]. For example, `/data/<modid
 Similarly, you may append to or override tags declared in other domains, such as Vanilla, by declaring your own JSONs.
 For example, to add your own mod's saplings to the Vanilla sapling tag, you would specify it in `/data/minecraft/tags/blocks/saplings.json`, and Vanilla will merge everything into one tag at reload, if the `replace` option is false.
 If `replace` is true, then all entries before the json specifying `replace` will be removed.
-See the [Vanilla wiki][tags] for a description of the base syntax.
-
-Forge provides two extensions on the Vanilla syntax:
+Values listed that are not present will cause the tag to error unless the value is listed using an `id` string and `required` boolean set to false, as in the following example:
+
+```json
+{
+  "replace": false,
+  "values": [
+    "minecraft:gold_ingot",
+    "mymod:my_ingot",
+    {
+      "id": "othermod:ingot_other",
+      "required": false
+    }
+  ]
+}
+```
 
-* You may declare an `optional` array of the same format as the `values` array, but any values listed here that are not present will not cause the tag loading to error.
-This has been deprecated in favor of the vanilla method of specifying optional tag values.
-* You may declare a `remove` array of the same format as the `values` array. Any values listed here will be removed from the tag. This acts as a finer grained version of the Vanilla `replace` option.
+See the [Vanilla wiki][tags] for a description of the base syntax.
+There is also a Forge extension on the Vanilla syntax. You may declare a `remove` array of the same format as the `values` array. Any values listed here will be removed from the tag. This acts as a finer grained version of the Vanilla `replace` option.
```

Comment:
New line mean replace the double space with a new line, not just remove the double space altogether without adding the new line.

Reply:
Instructions were unclear.  Had removed this double spacing: https://github.com/D0CTOR-ZED/Documentation/commit/7f46947f2c519b700fbbeba25eb8b11352e63cc9#diff-8c6dfe5a26f71f428c40a4e8e37dbfa99968d6ee1ad2d6e224f14c112d345a19L26

Had interpreted this, https://github.com/MinecraftForge/Documentation/pull/401#discussion_r791869271 , as meaning that you wanted the parts together.  Re-separated them.  Assuming this is the new line you wanted.